frontend/src/components/UserDetail/userDetail.module.css

```diff
@@ -12,9 +12,29 @@
   box-shadow: 0 0.25rem 2.2rem rgba(0, 0, 0, 0.1);
   padding: 1.875rem 3rem;
 }
+.pageDivTitle {
+  text-align: left;
+  font-size: 1.75rem;
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2.25rem 2.5rem;
+  font-weight: bold;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: inherit;
+  font-size: 1.75rem;
+  margin-bottom: 0;
+  margin-left: -28px;
```

Comment:
this is how we can fit the arrow into the `.pageDivTitle` padding. the arrow is 24px wide + 4px right margin, so we add a negative margin so all the content in `.header` shifts left 28px, leaving the "Students" or "Employees" text in the same place

Reply:
Sounds good, thanks for doing this.